geissonator/openbmc-events/openbmc-events

```diff
@@ -117,8 +120,12 @@ def do_clear_all_events(args):
 
 def do_list_dumps(args):
     s = BMC(server=args.server)
-    for e in s.list_dumps():
-        print(e)
+    try:
+        for e in s.list_dumps():
+            print(e)
+    except:
+        # ignore if we get nothing back
```

Comment:
This is a bit wild. If there are no dumps, we should just return an empty list, not throw an exception. Exceptions should be used for exceptional circumstances, and not having any dumps is not one.

Reply:
Makes sense, done